tools/__tests__/utils.test.ts

```
@@ -1,43 +1,40 @@
+    vol.fromJSON(volumeJson);
     vi.resetAllMocks();
+    io = makeIO();
+  });
+  describe("findRootDir", () => {
+    it("should find the root directory", () => {
+      const result = findRootDir("/saf/package.json", io);
+      expect(result).toBe("/saf");
+    });
   });
 
+  beforeEach(() => {});
```

Biome (1.9.4)

[error] Disallow duplicate setup and teardown hooks.

Disallow beforeEach duplicacy inside the describe function.

(lint/suspicious/noDuplicateTestHooks)